### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by igoradamenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,54 @@
 # Fitting
 
-<a href="https://funbox.ru">
```
Comment (igoradamenko) [inline README.md:3]: Replaced with the big one in the bottom of the README.

File: README.md [hunk 2]
```diff
@@ -75,30 +75,45 @@ Example:
...
-## Check documentation cover
+To match routes and validate JSON-schemas run:
 
-### xs size
+```bash
+rake fitting:documentation_responses[report_size]
+```
 
-For match routes and valid json-schemas run
+There are four types (or `report_size`) of reports available:
```
Comment (igoradamenko) [inline README.md:90]: I'm not sure that I got the meaning of the “sizes” correctly, so @tuwilof double-check it please.

File: README.md [hunk 3]
```diff
@@ -161,75 +174,79 @@ combination: ["required", "pages"]
...
 
-## Check tests cover
+Using this you can document API prefixes. 
+The task will create JSON (`fitting/report.json`) and HTML (`fitting/index.html`) reports.
 
-### xs size
+## Tests coverage
 
-`rake fitting:tests_responses[xs]`
+Only `xs` size is available for tests coverage:
```
Comment (igoradamenko) [inline README.md:190]: There were no other examples so I decided that only `xs` is available for tests coverage. Tell me if I wrong, I will fix it.

File: README.md [hunk 4]
```diff
@@ -161,75 +174,79 @@ combination: ["required", "pages"]
...
-For details `rake fitting:documentation_responses_error[m]`
-
-### l size
-
-In addition to the previous comand, you will learn the coverage(oneOf) json-schemas with task `rake fitting:documentation_responses[l]`
-For details `rake fitting:documentation_responses_error[l]`
+### Experimental report
 
-### new experimental report
+This report will be available and properly documented in the next major update, but you already can try it by running:
```
Comment (igoradamenko) [inline README.md:179]: @tuwilof and this should be checked too. Not sure that I got your explanation properly ^_^"

File: README.md (hunk 2 above)
Comment (tuwilof) [reply README.md:90]: yep, all right

File: README.md (hunk 4 above)
Comment (tuwilof) [reply README.md:179]: ok, the description is correct

File: README.md (hunk 3 above)
Comment (tuwilof) [reply README.md:190]: all right, only `xs`

**Pull request merged**